entoas/generator.go

```diff
@@ -541,6 +548,11 @@ func OgenSchema(f *gen.Field) (*ogen.Schema, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if ant.Skip {
+		return nil, nil
+	}
+
```

Comment:
I prefer to have this done more explicit by either removing the field [here](https://github.com/ent/contrib/blob/e738ef866e51c76d31fcc71919a7986d2e403719/entoas/generator.go#L59) or maybe [here](https://github.com/ent/contrib/blob/e738ef866e51c76d31fcc71919a7986d2e403719/entoas/generator.go#L127)

Reply:
Okey, I refactored code and made skip logic more explicit